### PR TITLE
DOC- 6270-C5-- addresses issues from Feedback Notes

### PR DESCRIPTION
**Pull request by ibsoln**

### CODE REVIEW

File: modules/ROOT/pages/_partials/supported-versions.adoc [hunk 1]
```diff
@@ -113,21 +114,24 @@ Comparing this to the xref:1.4@csharp.adoc#supported-versions[supported versions
 .Support Notice - Apple macOS
 CAUTION: macOS is supported ONLY for testing and development purposes.
 
-:msg_title: Apple macOS
-:msg_component: OSX 10.11
-:msg_release: 2.8
-include::{root-partials}deprecationNotice.adoc[]
+.Deprecated versions
```
Comment (amarantha-k) [inline modules/ROOT/pages/_partials/supported-versions.adoc]: Camel case "Versions"?

File: modules/objc/pages/learn/objc-fts.adoc [hunk 2]
```diff
@@ -1,132 +1,21 @@
-= Full Text Search Operations
+= Using Full-Text Search
 :page-layout: article
 :page-status:
 :page-edition:
 :page-role:
-:description: Couchbase mobile database full-text search concepts
+:description: Working with Couchbase Lite's data model  -- Querying database data using full text search
```
Comment (amarantha-k) [inline modules/objc/pages/learn/objc-fts.adoc]: reword "database data"?

**Review by amarantha-k — APPROVED** [review]
Please fix the two minor comments before merging.

File: modules/ROOT/pages/_partials/supported-versions.adoc (hunk 1 above)
Comment (ibsoln) [reply modules/ROOT/pages/_partials/supported-versions.adoc]: I thought table captions were sentence case?

File: modules/objc/pages/learn/objc-fts.adoc (hunk 2 above)
Comment (ibsoln) [reply modules/objc/pages/learn/objc-fts.adoc]: Change done locally  and force-pushed

File: modules/ROOT/pages/_partials/supported-versions.adoc (hunk 1 above)
Comment (ibsoln) [reply modules/ROOT/pages/_partials/supported-versions.adoc]: Change done locally  and force-pushed

**Pull request merged**